content/docs/user-guide/data-pipelines/index.md

```diff
@@ -0,0 +1,18 @@
+# Data Pipelines
+
+If you find yourself repeating sequence of actions to get or update the results
+of your project, then you may already have a pipeline. For example, a data
+science workflow could involve:
+
+1. Gathering data for training and validation
+2. Extracting useful features from the training dataset
+3. (Re)training an ML model
+4. Evaluating the results against the validation set
+
+DVC helps you [define] these stages in a standard YAML format (`dvc.yaml`),
```

Comment:
this kind of stages?

Reply:
Not sure why "kind of".

Reply:
otherwise it reads as more opinionated like _these_ specific stages from the list

Reply:
Gotcha. OK changed to "stages like these" 👍🏼